apps/ewallet_db/test/ewallet_db/token_test.exs

```diff
@@ -166,6 +168,62 @@ defmodule EWalletDB.TokenTest do
     end
   end
 
+  describe "query_all_blockchain/1" do
+    test "returns a query of tokens that have a blockchain address" do
+      assert Enum.empty?(Token.all())
+
+      :token |> params_for(%{blockchain_address: "0x01"}) |> Token.insert()
+      :token |> params_for(%{blockchain_address: "0x02"}) |> Token.insert()
+      :token |> params_for() |> Token.insert()
+
+      tokens = Token.query_all_blockchain() |> Repo.all()
+
+      assert length(tokens) == 2
```

Comment:
Can you assert blockchain address too? Just in case the 2 happen to be non blockchain.